src/performance/metrics/StatisticCollector.cpp

```diff
@@ -13,19 +13,21 @@ limitations under the License.
 
 #include "StatisticCollector.h"
 
-static clock_t lastCPU, lastSysCPU, lastUserCPU;
 static int numProcessors;
 
+static long parseLine(char *line);
+static long getSwapSpace(const char *type);
+
 int StatisticCollector::init() {
-    FILE* file;
+    FILE *file;
     struct tms timeSample;
     char line[128];
 
-    lastCPU = times(&timeSample);
-    lastSysCPU = timeSample.tms_stime;
-    lastUserCPU = timeSample.tms_utime;
-
     file = fopen("/proc/cpuinfo", "r");
+    if (!file) {
+        std::cout << "Cannot open /proc/cpuinfo" << std::endl;
```

Comment:
Please move this to a standard error log line.

Reply:
Done